mmv1/third_party/terraform/services/securitycenterV2/resource_scc_notification_config_test.go

```diff
@@ -0,0 +1,104 @@
+package securitycenter_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/client"
+	scc "google.golang.org/api/securitycenter/v1"
+)
+
+func TestAccSecurityCenterOrganizationNotificationConfig_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":        envvar.GetTestOrgFromEnv(t),
+		"topic_name":    acctest.RandomString(10),
+		"config_id":     acctest.RandomString(10),
+		"random_suffix": acctest.RandomString(10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy: testAccCheckSecurityCenterOrganizationNotificationConfigDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSecurityCenterOrganizationNotificationConfig_basic(context),
+			},
+			{
+				ResourceName: "google_scc_organization_notification_config.default",
+				ImportState: true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"parent",
+					"config_id",
+				},
+			},
+		},
+	})
+}
+
+func testAccSecurityCenterOrganizationNotificationConfig_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_pubsub_topic" "scc_organization_notification" {
```

Comment:
I think these need to be updated to use `google_scc_v2_organization_notification_config`, and you might need to update the import as well `scc "google.golang.org/api/securitycenter/v1"`

Reply:
addressed